lib/rules/no-attrs-in-components.js

```diff
@@ -8,7 +8,7 @@ const componentTemplateRegex = new RegExp(
 
 module.exports = class NoAttrsInComponents extends Rule {
   isComponentTemplate() {
-    return componentTemplateRegex.test(this._moduleName);
+    return componentTemplateRegex.test(this._filePath);
```

Comment:
I think this will need to fallback (triggering the deprecation) to avoid being breaking

```suggestion
    return componentTemplateRegex.test(this._filePath || this._moduleName);
```